index.js

```diff
@@ -127,8 +154,9 @@ NodePreGypGithub.prototype.publish = function(options) {
 		if(err) {console.error(err); return;}
 		
 		release	= (function(){ // create a new array containing only those who have a matching version.
+			var tag_name = options.tag_name || this.package_json.version;
```

Comment:
this is not needed. see [next comment](https://github.com/bchr02/node-pre-gyp-github/pull/11/files#diff-168726dbe96b3ce427e7fedce31bb0bcR159).